src/Look/HorizonLook.cpp

```diff
@@ -14,4 +14,7 @@ HorizonLook::Initialise()
 
   terrain_brush.Create(terrain_color);
   terrain_pen.Create(Layout::Scale(1), DarkColor(terrain_color));
+
+  no_data_font.Load(FontDescription(Layout::FontScale(22)));
```

Comment:
You copied that from the FlarmTrafficLook. That's bad. Don't submit duplicate code! This is not only duplicate code, but also adds resource usage because the font is loaded twice. Find a better solution to avoid code/data duplication.

Reply:
I'm not quite sure what to do here. There are two options that come to mind:
1. go for inline in the required function e.g.,:
`const unsigned no_data_font_size = Layout::FontScale(12);`
2. or somehow create a shared statement between horizon and traffic

I don't like two and I'm not sure if 1 is better than the current approach.

Are there other options, just a link to an example that I can see or is 1 preferable?

Reply:
@MaxKellermann just checking around your git commit suggestion - are you suggesting to do this in my current branch but just keep them as separate commits or something else, like insert this commit earlier in the sequence of commits?

Reply:
Yes, insert new commit in this branch. (If you don't know stgit - get it, it helps a lot with managing proper PR branches.)

Reply:
Hi @MaxKellermann 

I've struggled with implementing your suggestion. I just wanted to check my understanding of your suggestion is this:
 1. Create a new struct called GaugeLook
 2. Move the "no data" or "no traffice" font creation into GaugeLook e.g.,
 `no_data_font.Load(FontDescription(Layout::FontScale(22)));`
 3. In the FlarmTrafficLook.cpp create a variable called no_traffic_font that points to the no_data_font in GaugeLook.
 4. Then do this for HorizonLook

 Did I follow this correctly?

Reply:
I've committed my WIP change to so you can see my attempt. This obviously doesn't work and is broken but I thought it might be helpful for the conversation.